Pod/Classes/BONTag.m

```diff
@@ -0,0 +1,235 @@
+//
+//  BONTag.m
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import "BONTag_Private.h"
+#import "BONMot.h"
+
+static const NSString *kBONTagDefaultStartPrefix = @"<";
```

Comment:
#req the way these constants are defined, they can be redefined at run time without a compiler error. To get a compiler error if you try to reassign it, it should be:

```objc
static NSString * const kBONTagDefaultStartPrefix = @"<";
```

(`clang-format` may disagree about the spacing, which is fine.)